src/modules/admin/sections/ProfileEditor/ProfileEditor.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useEffect, useState } from 'react';
+import { useUser } from '@src/common/contexts';
+import { useTranslation } from 'react-i18next';
+
+import { Text, Button, Textarea } from '@common/components';
+import {
+  Avatar,
+  Stack,
+  HStack,
+  VStack,
+  Input,
+  Box,
+  SkeletonCircle,
+  SkeletonText,
+} from '@chakra-ui/react';
+
+export default function ProfileEditor() {
+  const { t } = useTranslation('admin');
+  const {
+    user: { username },
+  } = useUser();
+  const [profileTitle, setProfileTitile] = useState('');
+  const [bio, setBio] = useState('');
+  useEffect(() => {
+    setProfileTitile(username);
+  }, [username]);
+
+  if (!profileTitle)
+    return (
+      <Box minWidth="670px" padding="6" boxShadow="lg" bg="white">
+        <SkeletonCircle size="10" />
+        <SkeletonText mt="4" noOfLines={4} spacing="4" />
+      </Box>
+    );
+
+  return (
+    <Stack>
+      <Text fontSize="20px" fontWeight="600" alignSelf="flex-start">
```

Comment:
Try to modify the `Text` component so it returns a `heading3` variant, instead of changing the style here.

Reply:
Fixed: added 'h3' variant to `Text` component